be/src/exprs/agg/nullable_aggregate.h

```diff
@@ -349,9 +349,11 @@ class NullableAggregateFunctionUnary final
                     // all null
                     if constexpr (!IgnoreNull) {
                         for (size_t i = offset; i < offset + batch_nums; i++) {
-                            this->data(states[i] + state_offset).is_null = false;
-                            this->nested_function->process_null(
-                                    ctx, this->data(states[i] + state_offset).mutable_nest_state());
+                            if (!selection[i]) {
```

Comment:
Add one test case to unit test or Integration test?

Reply:
> I will add it later